<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfCertDialog</class>
 <widget class="QDialog" name="ConfCertDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Retroshare node details</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images.qrc">
    <normaloff>:/images/logo/logo_16.png</normaloff>:/images/logo/logo_16.png</iconset>
  </property>
  <layout class="QGridLayout" name="ConfCertDialog_GL">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="horizontalSpacing">
    <number>6</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="HeaderFrame" name="headerFrame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="frame_GL">
      <item row="1" column="0">
       <layout class="QGridLayout" name="buttons_GL">
        <item row="0" column="0">
         <widget class="QDialogButtonBox" name="buttonBox">
          <property name="standardButtons">
           <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QTabWidget" name="stabWidget">
        <property name="minimumSize">
         <size>
          <width>638</width>
          <height>0</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="detail_Tab">
         <attribute name="icon">
          <iconset resource="../images.qrc">
           <normaloff>:/images/info16.png</normaloff>:/images/info16.png</iconset>
         </attribute>
         <attribute name="title">
          <string>Details</string>
         </attribute>
         <layout class="QVBoxLayout" name="detail_Tab_VL">
          <item>
           <layout class="QHBoxLayout" name="detail_Tab_Top_HL">
            <item>
             <widget class="QGroupBox" name="detail_GBox">
              <property name="title">
               <string>Node info:</string>
              </property>
              <layout class="QGridLayout" name="detail_Tab_Left_GL">
               <item row="3" column="1">
                <widget class="QLineEdit" name="statusline">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="RSTextBrowser" name="pgpfingerprint">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>27</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_loc">
                 <property name="text">
                  <string>Node name :</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLineEdit" name="loc">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_status">
                 <property name="text">
                  <string>Status :</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="0">
                <widget class="QLabel" name="label_last_contact">
                 <property name="text">
                  <string>Last Contact :</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="1">
                <widget class="QLineEdit" name="lastcontact">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="6" column="0">
                <widget class="QLabel" name="label_version">
                 <property name="text">
                  <string>Retroshare version :</string>
                 </property>
                </widget>
               </item>
               <item row="6" column="1">
                <widget class="QLineEdit" name="version">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_id">
                 <property name="text">
                  <string>Node ID :</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="peerid">
                 <property name="toolTip">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This is the ID of the node's &lt;span style=&quot; font-weight:600;&quot;&gt;OpenSSL&lt;/span&gt; certifcate, which is signed by the above &lt;span style=&quot; font-weight:600;&quot;&gt;PGP&lt;/span&gt; key. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="label_crypto">
                 <property name="text">
                  <string>Encryption</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QLineEdit" name="crypto_info">
                 <property name="toolTip">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This is the encryption method used by &lt;span style=&quot; font-weight:600;&quot;&gt;OpenSSL&lt;/span&gt;. The connection to friend nodes&lt;/p&gt;&lt;p&gt;is always heavily encrypted and if DHE is present the connection further uses&lt;/p&gt;&lt;p&gt;&amp;quot;perfect forward secrecy&amp;quot;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_name">
                 <property name="text">
                  <string>Name:</string>
                 </property>
                </widget>
               </item>
               <item row="7" column="1">
                <widget class="QLineEdit" name="statusmessage"/>
               </item>
               <item row="7" column="0">
                <widget class="QLabel" name="label_status_message">
                 <property name="text">
                  <string>Status message:</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="detail_Tab_Right_VL">
              <item>
               <widget class="AvatarWidget" name="avatar">
                <property name="minimumSize">
                 <size>
                  <width>128</width>
                  <height>128</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>128</width>
                  <height>128</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="detail_Tab_Right_VL_Spacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="detail_Tab_VSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="connectivity_Tab">
         <attribute name="title">
          <string>Connectivity</string>
         </attribute>
         <layout class="QVBoxLayout" name="connectivity_Tab_VL">
          <item>
           <widget class="QGroupBox" name="currentAdd_GBox">
            <property name="title">
             <string>Current address:</string>
            </property>
            <layout class="QVBoxLayout" name="currentAdd_GBox_VL">
             <item>
              <layout class="QGridLayout" name="currentAdd_GL">
               <item row="1" column="3">
                <widget class="QSpinBox" name="localPort">
                 <property name="minimum">
                  <number>0</number>
                 </property>
                 <property name="maximum">
                  <number>65535</number>
                 </property>
                 <property name="value">
                  <number>7812</number>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QLineEdit" name="dynDNS"/>
               </item>
               <item row="2" column="3">
                <widget class="QSpinBox" name="extPort">
                 <property name="minimum">
                  <number>0</number>
                 </property>
                 <property name="maximum">
                  <number>65535</number>
                 </property>
                 <property name="value">
                  <number>7812</number>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="l_dynDNS">
                 <property name="text">
                  <string>Dynamic DNS</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="localAddress"/>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="l_extAddress">
                 <property name="text">
                  <string>External Address</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLineEdit" name="extAddress"/>
               </item>
               <item row="2" column="2">
                <widget class="QLabel" name="l_portExternal">
                 <property name="text">
                  <string>Port</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QLabel" name="l_portLocal">
                 <property name="text">
                  <string>Port</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="l_localAddress">
                 <property name="layoutDirection">
                  <enum>Qt::LeftToRight</enum>
                 </property>
                 <property name="text">
                  <string>Local Address</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="knownAdd_GBox">
            <property name="title">
             <string>List of known addresses:</string>
            </property>
            <layout class="QVBoxLayout" name="knownAdd_GBox_VL">
             <item>
              <widget class="QListWidget" name="ipAddressList">
               <property name="contextMenuPolicy">
                <enum>Qt::DefaultContextMenu</enum>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::MultiSelection</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="rs_cert_Tab">
         <attribute name="title">
          <string>Retroshare Certificate</string>
         </attribute>
         <layout class="QVBoxLayout" name="rs_cert_Tab_VL">
          <item>
           <widget class="QTextEdit" name="userCertificateText"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="rs_cert_CB_HL">
            <item>
             <widget class="QCheckBox" name="_shouldAddSignatures_CB">
              <property name="text">
               <string>Include signatures</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="_shortFormat_CB">
              <property name="text">
               <string>short format</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="_includeIPHistory_CB">
              <property name="text">
               <string>Include IP history</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="rs_cert_CB_HL_HSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RSTextBrowser</class>
   <extends>QTextBrowser</extends>
   <header>gui/common/RSTextBrowser.h</header>
  </customwidget>
  <customwidget>
   <class>HeaderFrame</class>
   <extends>QFrame</extends>
   <header>gui/common/HeaderFrame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AvatarWidget</class>
   <extends>QLabel</extends>
   <header>gui/common/AvatarWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>stabWidget</tabstop>
  <tabstop>pgpfingerprint</tabstop>
  <tabstop>peerid</tabstop>
  <tabstop>loc</tabstop>
  <tabstop>statusline</tabstop>
  <tabstop>crypto_info</tabstop>
  <tabstop>lastcontact</tabstop>
  <tabstop>version</tabstop>
  <tabstop>statusmessage</tabstop>
  <tabstop>userCertificateText</tabstop>
  <tabstop>_shouldAddSignatures_CB</tabstop>
  <tabstop>_shortFormat_CB</tabstop>
  <tabstop>_includeIPHistory_CB</tabstop>
 </tabstops>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
